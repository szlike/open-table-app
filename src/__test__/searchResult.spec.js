import React from "react";
import { create } from "react-test-renderer";


describe("SearchResult component", () => {
  test("Matches the snapshot", () => {
    const SearchResult = create(<searchResult />);
    expect(SearchResult.toJSON()).toMatchSnapshot();
  });
});

describe("Button component", () => {
    test("it shows the expected text when clicked (testing the wrong way!)", () => {
      const component = create(<Button text="SUBSCRIBE TO BASIC" />);
      const instance = component.getInstance();
      expect(instance.state.text).toBe("");
    });
  });
