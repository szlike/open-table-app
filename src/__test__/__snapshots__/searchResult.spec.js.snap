// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResult component Matches the snapshot 1`] = `<searchResult />`;
